language/libra-vm/src/libra_vm.rs

```diff
@@ -62,8 +69,15 @@ impl LibraVM {
         self.load_configs_impl(&RemoteStorage::new(state))
     }
 
+    fn on_chain_config(&self) -> VMResult<&OnlineConfig> {
+        self.on_chain_config
+            .as_ref()
+            .ok_or_else(|| VMStatus::new(StatusCode::VM_STARTUP_FAILURE))
+    }
+
     fn load_configs_impl(&mut self, data_cache: &dyn RemoteCache) {
         self.gas_schedule = self.fetch_gas_schedule(data_cache).ok();
+        self.on_chain_config = OnlineConfig::load_on_chain_config(data_cache);
```

Comment:
Should we panic or error out if this fails? It's not like we can do much otherwise...

Reply:
I agree--I think options are dangerous for both this and the gas schedule... Might be a subject for future refactoring though.